src/test/java/org/apache/sling/commons/mime/internal/MimeTypeServiceImplTest.java

```diff
@@ -152,6 +165,29 @@ public void testProvider2() throws Exception {
         assertNull(this.service.getMimeType(GIF));
     }
 
+    public void testMultipleMimeTypes() throws Exception {
+        loadMimeTypes(MimeTypeServiceImpl.CORE_MIME_TYPES);
+        loadMimeTypes(MimeTypeServiceImpl.MIME_TYPES);
+
+        for (String mm : epsMimeTypeExt.keySet()) {
+            assertEquals("Extension " + mm + " (1)", epsMimeTypeExt.get(mm), this.service.getExtension(mm));
```

Comment:
The name _mm_ is a bit opaque to me, can you please use something to understand?

Reply:
agreed, changing it to extension